test/helper/webapi.js

```diff
@@ -652,10 +652,10 @@ module.exports.tests = function () {
       await I.amOnPage('/form/file');
       await I.attachFile('#avatar', 'app/avatar.jpg');
       await I.click('Submit');
-      await I.amOnPage('/');
+      await I.see('Thank you');
       formContents().files.should.have.key('avatar');
       formContents().files.avatar.name.should.eql('avatar.jpg');
-      formContents().files.avatar.type.should.eql('image/jpeg');
+      isHelper('Playwright') ? formContents().files.avatar.type.should.eql('application/octet-stream') : formContents().files.avatar.type.should.eql('image/jpeg');
```

Comment:
@PeterNgTr do you have any ideas why this is not image/jpeg? Probably the file is not actually read?

Reply:
@DavertMik it is actually read.

```sh
  Playwright
    #attachFile
{
  avatar: {
    name: 'avatar.jpg',
    type: 'application/octet-stream',
    tmp_name: '/private/var/folders/xs/tsgkwpzs3qncr57vzktsn7lw0000gn/T/phpJtR2zZ',
    error: 0,
    size: 5621
  }
}
      ✓ should upload file located by CSS (1636ms)
```

Reply:
yeah, I created an issue in Playwright. Looks like a their issue

Reply:
this issue is fixed, we will definitely update our tests.

Reply:
They are fast! Nice!